src/backend/migrations/Migrations.h

```diff
@@ -0,0 +1,37 @@
+#ifndef MIGRATION_TEMPLATE_H_INCLUDED
```

Comment:
Your branch go so far behind `develop` 🤣 
You really need to rebase on latest `develop` asap.
We use `#pragma once` instead of this everywhere since a long time already.

Reply:
Also you will see when you rebase but we added a header comment with copyright info, like in rippled.